lib/ws_servers/api/index.js

```diff
@@ -126,13 +130,11 @@ module.exports = class APIWSServer extends WSServer {
       ws.aoc.close()
     }
 
-    removePoolClient(this.pc, ws)
-
     const subExchanges = Object.keys(ws.subscriptions || {})
 
     subExchanges.forEach((exID) => {
       ws.subscriptions[exID].forEach((channelData) => {
-        this.pc.send(['unsub', exID, channelData])
+        this.exchangeClient.unsubscribe(channelData)
```

Comment:
should we also call this.exchangeClient.close() ?

Reply:
hmmm, thought about this question:

no, not in this case. we maintain one open websocket connection that we open in the constructor of the server. 

in the next PR a watchdog will maintain that the connection stays open, even after the wifi disconnects, for as long as the server runs. its this bug that was the main reason for the refactor.